### PR TITLE
Refactor some parts of NewLaunch

### PR DESCRIPTION
**Pull request by icelimetea**

### CODE REVIEW

File: libraries/launcher/org/multimc/EntryPoint.java [hunk 1]
```diff
@@ -16,136 +16,123 @@
...
-            else if (pair[0].equals("abort"))
+            case "abort": {
                 return Action.Abort;
+            }
 
-            else throw new ParseException("Error while parsing:" + pair[0]);
-        }
+            case "launcher": {
+                if (pair.length != 2)
+                    throw new ParseException("Expected 2 tokens, got 1!");
```
Comment (Scrumplex) [inline libraries/launcher/org/multimc/EntryPoint.java]: this will also be thrown if there are three or more tokens

**Review by Scrumplex — APPROVED** [review]
LGTM! Thanks

File: libraries/launcher/org/multimc/EntryPoint.java (hunk 1 above)
Comment (icelimetea) [reply libraries/launcher/org/multimc/EntryPoint.java]: There are second argument in split method, which forces to split string only on two parts.

Comment (icelimetea) [reply libraries/launcher/org/multimc/EntryPoint.java]: I think I'll change it to output pair.length as amount of tokens, just for the clarity

Comment (Scrumplex) [reply libraries/launcher/org/multimc/EntryPoint.java]: > There are second argument in split method, which forces to split string only on two parts.

Oh, you are right. Might have missed that 😅

**Review by Scrumplex — APPROVED** [review]

**Review by timoreo22 — APPROVED** [review]

**Review by txtsd — APPROVED** [review]

**Pull request merged**